src/Runtime.php

```diff
@@ -4,20 +4,30 @@
 
 abstract class Runtime
 {
+    public const NODEJS_20 = 'nodejs20.x';
+
+    public const NODEJS_18 = 'nodejs18.x';
+
     public const NODEJS_16 = 'nodejs16.x';
 
     public const NODEJS_14 = 'nodejs14.x';
```

Comment:
Maybe add a deprecation warning comment here, so that people know to upgrade?
```suggestion
    /** @deprecated */
    public const NODEJS_14 = 'nodejs14.x';
```